configuration/authentication.mdx

```diff
@@ -87,6 +87,8 @@ authentication:
   methods:
     oidc:
       enabled: true
+      email_matches:
```

Comment:
we might want to add a bit more on this doc page about the `email_matches` functionality and what it's used for. ie: can be the email string or [regex](https://github.com/google/re2/wiki/Syntax) (`^.*@domain.com$`)